packages/block-library/src/post-comments-form/form.js

```diff
@@ -84,8 +84,7 @@ const CommentsForm = ( { postId, postType } ) => {
 		if ( 'closed' === commentStatus ) {
 			const actions = [
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen, while you add `core/post-comments-form` block, and you have `comments` disabled.

| Before | After     |
| :-------- | :------- |
| <img width="834" alt="Comments-form-before" src="https://github.com/user-attachments/assets/15579027-2e6a-4060-8119-19262dd9b24a"> | <img width="834" alt="Commentss-form-after" src="https://github.com/user-attachments/assets/088df1d9-92b3-488d-950a-2c07278345ad"> |